LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan/Hertz.cs

```diff
@@ -25,6 +25,8 @@ namespace LoRaWan
         public double Mega => this.value / 1e6;
         public double Giga => this.value / 1e9;
 
+        public static Hertz FromMega(double value) => new Hertz((ulong)(value * 1e6));
```

Comment:
This will trim. Do you want to round instead? Also, I'd recommend making a `checked` conversion:

```suggestion
        public static Hertz FromMega(double value) => new Hertz(checked((ulong)(value * 1e6)));
```

Reply:
I think trimming is fine. In the end we are going from a potential XXX.ZZZ value to a XXXZZZ000 value no?

Reply:
I guess it should be fine for since we multiple by a million before.